doc/plotting.rst

```diff
@@ -174,14 +174,42 @@ entire figure (as for matplotlib's ``figsize`` argument).
 
 .. _not equivalent to matplotlib's: https://github.com/mwaskom/seaborn/issues/746
 
+
+Multiple lines showing variation along a dimension
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to make line plots of two-dimensional data by calling ``xarray.plot.line()`` with appropriate arguments. Consider the 3D variable ``air``
+
+.. ipython:: python
+
+    air
```

Comment:
We use `air` as a running example on this page. I don't think we need to print it again here.